### PR TITLE
Add integration test of @skip and @include directives

### PR DESCRIPTION
**Pull request by no1seman**
This PR adds integration test of @skip and @include directives:

TEST($skip_field: Boolean){test{uri@skip(if: $skip_field) uris{uri}}}

TEST($skip_field: Boolean){test{uri@include(if: $skip_field) uris{uri}}}

Useful URLs:
https://spec.graphql.org/October2021/#sec--skip
https://spec.graphql.org/October2021/#sec--include
(https://graphql.org/learn/queries/ was useful for me to understand how it expected to work)

### CODE REVIEW

File: test/integration/graphql_test.lua [hunk 1]
```diff
@@ -1726,3 +1726,81 @@ function g.test_specifiedByURL_long_scalar()
...
+        nil,
+        nil,
+        { variables = { skip_field = true }}
+    )
+    t.assert_equals(errors, nil)
+    t.assert_items_equals(data, {test = {uris = {uri = "uri2"}}})
+
+    -- check request with directive: include == false
+    data, errors = check_request(
+        'query TEST($skip_field: Boolean){test{uri@include(if: $skip_field) uris{uri}}}',
```
Comment (DifferentialOrange) [inline test/integration/graphql_test.lua]:
```suggestion
        'query TEST($include: Boolean){test{uri@include(if: $include) uris{uri}}}',
```
or something like that

File: test/integration/graphql_test.lua [hunk 2]
```diff
@@ -1726,3 +1726,81 @@ function g.test_specifiedByURL_long_scalar()
...
+        query_schema,
+        nil,
+        nil,
+        { variables = { skip_field = false }}
+    )
+    t.assert_equals(errors, nil)
+    t.assert_items_equals(data, {test = {uris = {uri = "uri2"}}})
+
+    -- check request with directive: include == false
```
Comment (DifferentialOrange) [inline test/integration/graphql_test.lua]:
```suggestion
    -- check request with directive: include == true
```

File: test/integration/graphql_test.lua [hunk 3]
```diff
@@ -1726,3 +1726,81 @@ function g.test_specifiedByURL_long_scalar()
...
+    data, errors = check_request(
+        'query TEST($skip_field: Boolean){test{uri@include(if: $skip_field) uris{uri}}}',
```
Comment (DifferentialOrange) [inline test/integration/graphql_test.lua]:
```suggestion
        'query TEST($include: Boolean){test{uri@include(if: $include) uris{uri}}}',
```
or something like that

File: test/integration/graphql_test.lua [hunk 4]
```diff
@@ -1726,3 +1726,81 @@ function g.test_specifiedByURL_long_scalar()
...
+        query_schema,
+        nil,
+        nil,
+        { variables = { skip_field = false }}
```
Comment (DifferentialOrange) [inline test/integration/graphql_test.lua]:
```suggestion
        { variables = { include = false }}
```
or something like that

File: test/integration/graphql_test.lua [hunk 5]
```diff
@@ -1726,3 +1726,81 @@ function g.test_specifiedByURL_long_scalar()
...
+    t.assert_equals(errors, nil)
+    t.assert_items_equals(data, {test = {uris = {uri = "uri2"}}})
+
+    -- check request with directive: include == false
+    data, errors = check_request(
+        'query TEST($skip_field: Boolean){test{uri@include(if: $skip_field) uris{uri}}}',
+        query_schema,
+        nil,
+        nil,
+        { variables = { skip_field = true }}
```
Comment (DifferentialOrange) [inline test/integration/graphql_test.lua]:
```suggestion
        { variables = { include = true }}
```
or something like that

**Review by DifferentialOrange — COMMENTED** [review]
Some test variable names and comments are need to be fixed, but overall everything seems alright. Thank you for your contribution!

Also I think it would be nice to refer specification in PR description and commit message:
https://spec.graphql.org/October2021/#sec--skip
https://spec.graphql.org/October2021/#sec--include
(https://graphql.org/learn/queries/ was useful for me to understand how it expected to work)

**Review by DifferentialOrange — APPROVED** [review]

**Review by Totktonada — APPROVED** [review]
LGTM.

**Pull request merged**